test/e2e/storage_test.go

```diff
@@ -26,14 +28,22 @@ func testStorage(t *testing.T) {
 	// Create the PVC and choose the node the deployment will be scheduled on. This is necessary as ReadWriteOnly
 	// volumes can only be bound to a single node.
 	// https://docs.openshift.com/container-platform/4.12/storage/understanding-persistent-storage.html#pv-access-modes_understanding-persistent-storage
-	var pvcVolumeSource *core.PersistentVolumeClaimVolumeSource
-	pvc, err := tc.CloudProvider.CreatePVC(tc.client.K8s, tc.workloadNamespace)
-	require.NoError(t, err)
+	var pvc *core.PersistentVolumeClaim
+	if inTreeUpgrade {
```

Comment:
Can't we test this before waiting for Windows node and fail early when not on vSphere?